src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -253,4 +277,44 @@ public RegressionChecks generateTestChecks(ExecutableSequence sequence) {
     }
     return checks;
   }
+
+  /**
+   * Returns true if the method:
+   *
+   * <ul>
+   *   <li>has one formal parameter
+   *   <li>has a return type of primitive, boxed primitive, String, or enum
+   *   <li>is side-effect-free
```

Comment:
This does not seem to be true; at least, I don't see an implementation of this in the method body.  The method seems to already assume that any routine passed to it will be side-effect-free.